<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="mapper_awaits_before_reconnect" skipped="0" tests="1" time="9.44" timestamp="2022-06-02T08:47:14">
	<testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED"/>
		<system-out>08:47:04 INFO  ==============================================================
08:47:04 INFO  Id:    mapper_awaits_before_reconnect
08:47:04 INFO  Dir:   tedge_mapper_c8y/
08:47:04 INFO  Title: Validate mapper doesn't reconnect too often
08:47:04 INFO  ==============================================================
08:47:04 INFO  Started mosquitto_sub with process id 14585
08:47:04 INFO  Waiting for 0.1 seconds
08:47:05 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
08:47:05 INFO  Waiting for 5.0 seconds
08:47:10 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
08:47:13 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
08:47:13 WARN  Grep on tedge_mapper_journal.out contains &quot;ERROR&quot; ... failed [run.py:67]
08:47:13 INFO  Line count on file tedge_mapper_journal.out ... passed
08:47:13 WARN  Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0 ... failed [run.py:78]
08:47:13 WARN  Aborted test due to FAILED outcome
08:47:13 INFO  
08:47:14 INFO  Test duration: 9.44 secs
08:47:14 INFO  Test final outcome:  FAILED
08:47:14 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0
08:47:14 INFO  
</system-out>
	</testcase>
</testsuite>
